app/config.go

```diff
@@ -239,17 +247,20 @@ func InitGenesis() {
 			// if does not exist create one
 			jsonFile, err := os.OpenFile(genFP, os.O_RDWR|os.O_CREATE, os.ModePerm)
 			if err != nil {
-				panic(err)
+				fmt.Println(err)
+				os.Exit(1)
```

Comment:
```suggestion
                              log.Fatal(err)
```
stdlib package `log` offers `log.Fatalf` which logs the error with a timestamp & calls onto os.Exit(1)

the `log` package needs to also be added on this file for this suggestion to work